include/deal.II/lac/block_linear_operator.h

```diff
@@ -844,6 +914,64 @@ block_back_substitution(const BlockLinearOperator<Range, Domain> &block_operator
 
 //@}
 
+#ifdef DEAL_II_WITH_TRILINOS
+
+// Forward declarations:
+
+namespace TrilinosWrappers
+{
+  class BlockSparseMatrix;
+
+  namespace internal
+  {
+    namespace LinearOperator
+    {
+      class TrilinosPayload;
+    }
+
+    namespace BlockLinearOperator
+    {
+      template<typename PayloadBlockType>
+      class TrilinosBlockPayload;
+    }
+  }
+
+  // Specialisations for TrilinosWrappers::BlockSparseMatrix
+  // Note: The payloads should be correctly initialised by the standard
```

Comment:
This is a completely petty comment, but one I've learned the hard way: we almost uniformly use American English spelling, and that makes searching for particular text you know is somewhere easy. But here you use British English twice. May I gently suggest you change this?